baseline-error-prone/src/test/java/com/palantir/baseline/errorprone/ConsistentLoggerNameTest.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * (c) Copyright 2020 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.baseline.errorprone;
+
+import org.junit.jupiter.api.Test;
+
+class ConsistentLoggerNameTest {
+
+    @Test
+    void testFix() {
+        fix().addInputLines(
+                        "Test.java",
+                        "import org.slf4j.*;",
+                        "class Test {",
+                        "    private static final Logger LOG = LoggerFactory.getLogger(Test.class);",
```

Comment:
how about `logger` -> `log`? or do you think that's cool?

Reply:
Everything goes to `log`. There can only be 1 name for loggers